prac_05/emails.py

```diff
@@ -1,7 +1,7 @@
 """
 CP1404 EMAILS
 By Samantha Martin
-Used solutions for assistance in this program as I was having issues
+I used solutions for assistance in this program as I was having issues
 when I was splitting my email address.
 """
 
```

Comment:
Couldn't add comments to the code so I'll do it here. Lines 10-12 inclusive can be stored into a function for easier reading. EMAIL_NAME is a list that is changing with input so it can't be stored as a constant (not changing) with all capitals. Besides that, you've done a really good job here! :)